libs/content-search-toolkit/src/queries/tagQuery.ts

```diff
@@ -9,14 +9,13 @@ export const tagQuery = (tag: elasticTagField) => ({
       bool: {
         must: [
           {
-            term: {
-              'tags.key': tag.key,
-            },
-          },
-          {
-            term: {
-              'tags.type': tag.type,
-            },
+            term: tag.key
+              ? {
+                  'tags.key': tag.key,
+                }
+              : {
+                  'tags.value': tag.value,
+                },
```

Comment:
This is the elastic search tag query and is used for all tags in the main search engine not only limited to generic tags.
It seems to me this might break the current tag filtering in the main search engine, and also remove the ability to differentiate different types of tags.
Why do you want to change the query in this way?

Reply:
Removing the `type` filter was an accident :o But otherwise this was supposed to be fully compatible with current queries. The query should only change if the tag included a value instead of a key.

But you're right it's much simpler to just put the slug in the key of the tags. I have reverted this and related changes.